packages/serverpod/lib/src/server/server.dart

```diff
@@ -205,6 +211,16 @@ class Server {
       readBody = false;
     }
 
+    /// This [OPTIONS] check is necessary when making requests from
```

Comment:
Names in `[]` refers to other Dart code, so in this case should not be used. Also, the tripple dash comments are only used for Dart docs (here `//` double dashes should be used).